### PR TITLE
combined all managing functions on moderator panel

### PR DESCRIPTION
**Pull request by AhnessP**

### CODE REVIEW

File: EasyRestProjectNetTeam2/EasyRestPages/ModeratorManagePage.cs [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+        public void ClickUsersButton()
+        {
+            _usersButton.Click();
+        }
+
+        public void ClickOwnersButton()
+        {
+            _ownersButton.Click();
+        }
```
Comment (SerhiiTarasiuk) [inline EasyRestProjectNetTeam2/EasyRestPages/ModeratorManagePage.cs:87]: I've added this in Left Bar

File: EasyRestProjectNetTeam2/EasyRestPages/ModeratorManagePage.cs [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        
+        [FindsBy(How = How.XPath, Using = "//span[text()='Restaurants']")]
+        private IWebElement _restaurantsButton;
+        
+        [FindsBy(How = How.XPath, Using = "//span[text()='Users']")]
+        private IWebElement _usersButton;
+
+        [FindsBy(How = How.XPath, Using = "//span[text()='Owners']")]
+        private IWebElement _ownersButton;
```
Comment (SerhiiTarasiuk) [inline EasyRestProjectNetTeam2/EasyRestPages/ModeratorManagePage.cs:21]: I've added this in Left Bar

**Review by eugenek7 — APPROVED** [review]

**Review by Slim1011 — APPROVED** [review]

File: EasyRestProjectNetTeam2/EasyRestPages/ModeratorManagePage.cs [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        public void ClickRestoreRestaurantButton()
+        {
+            _restoreRestaurantButton.Click();
+        }
+
+        public void ClickUndoActionPopUpButton()
+        {
+            _undoActionPopUpButton.Click();
+        }
+
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/ModeratorManagePage.cs]: please remove the useless lines

**Review by dmatsev — APPROVED** [review]
Please resolve the comment before merging. Aprrove.

**Pull request merged**